en/advanced_features/satcom.md

```diff
@@ -0,0 +1,221 @@
+
+###### Module
+The module can either use the internal antenna or an external one connected to the SMA connector. To [switch between the two antennas modes](https://docs.rockblock.rock7.com/docs/switching-rockblock-9603-antenna-mode) the position of a small RF link cable needs to changed. If an external antenna is used always make sure that the antenna is connected to the module before powering it up to avoid damage to the module.
+
+The default baud rate of the module is 19200. However, the iridiumsbd driver of the PX4 Autopilot requires a baud rate of 115200 so it needs to be changed using the [AT commands](http://www.rock7mobile.com/downloads/IRDM_ISU_ATCommandReferenceMAN0009_Rev2.0_ATCOMM_Oct2012.pdf).
+
+1. Connect to the module with using a 19200/8-N-1 setting and check if the communication is working using the command: `AT`. The response should be: `OK`.
+2. Change the baud rate:
+
+    `AT+IPR=9`
+
+3. Reconnect to the model now with a 115200/8-N-1 setting and save the configuration using
+
+    `AT&W0`
+
+Now the module is ready to be used together with the PX4 autopilot.
+
+###### Software
+Flash the Pixhawk with the latest PX4 master. To activate either set the `SYS_COMPANION` parameter to `Iridium Telemetry (115200 baud, 8N1)` or add the following lines to the `extras.txt` on the SD card:
+
+```
+	# add a sleep here to make sure that the module is powered
+	usleep 200000
+
+	#driver
+	if iridiumsbd start -d SERIAL_PORT
+	then
+		# start the iridium mavlink instance
+		mavlink start -d /dev/iridium -m iridium -b 115200
+
+		# Sleep a bit to give Mavlink time to set up
+		usleep 100000
+
+	else
+		echo "WARN: Iridiumsbd driver not started, reboot"
+		tune_control play -m "ML<<CP4CP4CP4CP4CP4"
+	fi
+```
+
+where `SERIAL_PORT` is the serial port to which the RockBlock module is connected.
+
+#### RockBlock
+When buying the first module on RockBlock an user account needs to be created in a first step. Log in to the [account](https://rockblock.rock7.com/Operations) and register the RockBlock module under the `My RockBLOCKs`. Activate the line rental for the module and make sure that enough Credits for the expected flight duration are available on the account. When using the default settings one message per minute is sent from the airplane to the ground station.
+
+Set up a delivery group for the message relay server and add the module to that delivery group:
+
+![Delivery Groups](../../assets/satcom/deliverygroup.png)
+
+#### Relay Server
+It is expected that the relay server is either using the Ubuntu 16.04 or 14.04 OS. 
+* The server working as a message relay should have a static IP address and two publicly accessible, open TCP ports:
+
+    *  `5672` for the RabbitMQ message broker (can be changed in the rabbitmq settings)
+    * `45679` for the HTTP POST interface (can be changed in the `relay.cfg` file)
+
+* Install the required python modules:
+
+```
+    sudo pip install pika
```

Comment:
Reduce to one line: `sudo pip install pika tornado future`?

Reply:
Ok, changed.